pyp2rpm/metadata_extractors.py

```diff
@@ -300,7 +301,8 @@ def runtime_deps(self):  # install_requires
         if self.metadata['entry_points'] and 'setuptools' not in install_requires:
             install_requires.append('setuptools')  # entrypoints
 
-        return self.name_convert_deps_list(deps_from_pyp_format(install_requires, runtime=True))
+        return self.name_convert_deps_list(deps_from_pyp_format(
+            sorted(install_requires), runtime=True))
```

Comment:
`install_requires` can also contain smth like `alabaster<0.8,>=0.7`, which after converting from pyp format transforms into 
```
Requires:       python3-alabaster < 0.8
Requires:       python3-alabaster >= 0.7
```
And as we sort before converting, those are still randomly switching places. Maybe we could do sorting after or while converting?